### ISSUE TITLE
README and INSTALL update for github

### ISSUE THREAD

**Issue by jd-moulton** [amanzi/amanzi#37]
The README needs to be updated to reflect

- repo moved to github
- external collaborators can use pull requests
- if significant collaboration is planned then copyright assignment or assertion is needed

The INSTALL needs to be updated to reflect the location, and we need to get the UserGuide on line as quickly as possible.

**Comment by smolins** [comment]
I would also suggest renaming (or adding a) README.md and using github markdown for nicer landing page for the repo.

**Comment by jd-moulton** [comment]
Yes, I agree this will be included in the updates here.   Although, the real landing "page" for non-developers will be the Sphinx-based front page that will provide links to downloads, documentation, etc.
